wpe/src/main/java/com/wpe/wpeview/WPEView.java

```diff
@@ -31,6 +31,9 @@ public class WPEView extends FrameLayout implements PageObserver {
     private final Context context;
     private final WPESettings settings = new WPESettings();
 
+    private static int ID = 0;
```

Comment:
If it is not a constant, it should be written in lower-case.

To be more understandable, maybe something like *pageId* or *nextPageId*.

I'm not sure 100% but if pages can be created from different threads, then this variable may need to be *volatile* to ensure correct synchronization.

Reply:
good point. Removed ID field completely as we don't need it anymore